lib/electric_slide/call_queue.rb

```diff
@@ -54,6 +65,14 @@ def get_agent(id)
     # Registers an agent to the queue
     # @param [Agent] agent The agent to be added to the queue
     def add_agent(agent)
+      case @connection_type
+      when :call
+        # FIXME: We want this to raise in the caller, and not kill the Queue actor
```

Comment:
I would expect the strack trace to include a reference to the line of code that failed. In the above stack trace, the class failing to initialize is nowhere to be seen.